Compiling.md

```diff
@@ -5,23 +5,42 @@ permalink: a/compiling/
 article: true
 ---
 
+A lot changed from ELMAH 1x to ELMAH 2x, why this document is split into two parts.
+
+## ELMAH 2x
+
+You can compile ELMAH from the sources in one of two ways:
```

Comment:
I think we need to simplify this whole area. It made sense to talk about building from the command line & VS previously when VS solutions were not MSBuild files.